src/runtime/runtime_tinygowasm_unknown.go

```diff
@@ -0,0 +1,50 @@
+//go:build wasm_unknown
+
+package runtime
+
+const (
+	stdout = 1
+)
+
+func putchar(c byte) {
+}
+
+func getchar() byte {
+	// dummy, TODO
+	return 0
+}
+
+func buffered() int {
+	// dummy, TODO
+	return 0
+}
+
+//go:linkname now time.now
+func now() (sec int64, nsec int32, mono int64) {
+	return 0, 0, 0
+}
+
+// Abort executes the wasm 'unreachable' instruction.
+func abort() {
+	trap()
+}
+
+//go:linkname syscall_Exit syscall.Exit
+func syscall_Exit(code int) {
+	//proc_exit(uint32(code))
```

Comment:
You could trap for a non-zero exit code. Then it will stop code execution, as a syscall.Exit is expected to do. On the downside, it won't execute defers.

Reply:
Removed commented out code.